compair/api/comparison_example.py

```diff
@@ -47,14 +50,14 @@ def post(self, course_uuid, assignment_uuid, comparison_example_uuid):
             answer1.practice = True
             comparison_example.answer1 = answer1
         else:
-            return {"error": "Comparison examples must have 2 answers"}, 400
+            abort(400, title="Comparison Example Update Failed", message="Comparison examples must have 2 answers.")
 
         if answer2_uuid:
             answer2 = Answer.get_active_by_uuid_or_404(answer2_uuid)
             answer2.practice = True
             comparison_example.answer2 = answer2
         else:
-            return {"error": "Comparison examples must have 2 answers"}, 400
+            abort(400, title="Comparison Example Update Failed", message="Comparison examples must have 2 answers.")
```

Comment:
"Comparison Example Not Updated" / "Please add two answers to the comparison example and try again."